apps/web/components/eventtype/EventAvailabilityTab.tsx

```diff
@@ -52,43 +54,28 @@ const SingleValue = ({ ...props }: SingleValueProps<AvailabilityOption>) => {
 
 const AvailabilitySelect = ({
   className = "",
+  options,
+  value,
   ...props
 }: {
   className?: string;
   name: string;
-  value: number;
+  value: AvailabilityOption | undefined;
+  options: AvailabilityOption[];
+  isDisabled?: boolean;
   onBlur: () => void;
   onChange: (value: AvailabilityOption | null) => void;
 }) => {
-  const { data, isLoading } = trpc.viewer.availability.list.useQuery();
   const { t } = useLocale();
-
```

Comment:
Moved one level up to be able to manipulate the options available and know it beforehand to show needed differences in the UI